plugins/manual_maps/ManualMapLoading.ts

```diff
@@ -0,0 +1,214 @@
+import fs from 'fs/promises'
+
+import { Logger } from '../../src/Logger.js'
+import { Client } from '../../src/client/Client.js'
+import config from '../../config/Config.js'
+import { MapService } from "../../src/services/MapService.js"
+
+type ReturnCode = {
+  code: number
```

Comment:
delete this while you still can for the love of our lord and saviour jesus christ